.storybook/main.js

```diff
@@ -63,9 +62,10 @@ module.exports = {
               // sourceMap: argv.sourcemap,
             },
           },
-        ],
-      },
-    ]);
+        ].filter(Boolean),
```

Comment:
Clever, I like it a lot. Maybe a ternary on `isEnvProduction` is clearer, though. Both for us and the JIT of the VM. :)

Reply:
I've taken it from CRA actually - it's prevalent in their webpack.config.js